src/table/headerRenderers/dataHeader.component.test.tsx

```diff
@@ -13,7 +13,15 @@ describe('Data Header', () => {
   const onClose = jest.fn();
 
   const createView = (): RenderResult => {
-    return render(<DataHeader {...props} />);
+    return render(
+      <table>
+        <thead>
+          <tr>
+            <DataHeader {...props} />
+          </tr>
+        </thead>
+      </table>
+    );
```

Comment:
This tests the column header in a proper tabular context while also removing the error of invalid DOM nesting in snapshot tests